test/parallel/test-runner-run.mjs

```diff
@@ -465,5 +466,18 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
           code: 'ERR_INVALID_ARG_TYPE'
         }));
     });
+
+    it('should pass instance of stream to setup', async () => {
```

Comment:
If no files passed I don't think the setup would be called

Reply:
I think, it will. Anyway, I have updated the test to run with a file. also, committed suggestion.